index.html

```diff
@@ -1,19 +1,116 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<body class="container">
+  <nav>
+    <div class="link-container">
+      <a href="services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </div>
+    <div class="great-idea">
+      <img class="computer-logo" src="img/logo.png" alt="Great Idea! Company logo." />
+    </div>
+  </nav>
+
+  <header>
+    <div class="headerText">
+      <h1>
+        Innovation <br>
+        On<br>
+        Demand
+      </h1>
+      <button>Get Started</button>
+    </div>
+    <div>
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+
+  </header>
+
+
+
+  <section class="topSection">
+    <div class="features">
+      <h2>Features</h2>
+      <p>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </p>
+    </div>
+
+    <div class="about">
+      <h2>About</h2>
+      <p>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </p>
+    </div>
+    <div>
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
```

Comment:
You want to move `middle-mg` class up to the div container, not an image, because it will flow out of the section. After You do this, You want to move whole div for middle image outside the about section so it is between bottom and about.